config/initializers/init_listeners.rb

```diff
@@ -0,0 +1,7 @@
+# Be sure to restart your server when you modify this file.
+
+unless defined?(::Rails::Console) || ENV['SCHED_TASK']
```

Comment:
@hsong-rh This is confusing, shouldn't the || be &&. You want to run that section only if both the values are not defined.

Reply:
👍

Reply:
OK, changed it into more clear format